### ISSUE TITLE
esp32c3phase.ino:189:17: error: 'median_add' was not declared in this scope

### ISSUE THREAD

**Issue by TwentySixer** [bertrik/MainsFrequency#3]
The median_add function is not defined.

`esp32c3phase.ino:189:17: error: 'median_add' was not declared in this scope`

The build fails.

**Comment by bertrik** [comment]
You're right. This repository in its current state contains some experiments that are not in shape.

I've created a new project based on the esp32c3phase code:
https://github.com/bertrik/GridFrequency

Please consider this code outdated, I might remove it altogether now.
